conf/i18n/en/my-account.yaml

```diff
@@ -92,3 +92,28 @@ changePassword:
   changeBtn: Change password
   backBtn: Back
   feedbackMessage: Password changed successfully.
+resetPassword:
+  title: Reset Password
+  newPasswordLabelTxt: New password
+  confirmPasswordLabelTxt: Confirm new password
+  resetBtn: Reset password
+  feedbackMessage: Password reset successfully.
+forgotPassword:
+  title: Forgot Password
+  description: Please submit your email address to receive a password reset link.
+  email:
```

Comment:
The email is rendered via the handlebar template `password-reset-email` and uses the locale of the request. But you're right that I forgot to add the i18n entries for german to the `de/my-account.yaml` file ;-)